### ISSUE TITLE
Exception in console when loosing focus

### ISSUE THREAD

**Issue by LDAP** [daveleroy/sublime_debugger#209]
Whenever ST looses focus, the console logs an exception:

```
Traceback (most recent call last):
  File "/opt/sublime_text/Lib/python38/sublime_plugin.py", line 1477, in is_enabled_
    ret = self.is_enabled(**args)
  File "/home/lucas/.config/sublime-text/Packages/Debugger/modules/command.py", line 104, in is_enabled
    command = CommandsRegistry.commands_by_action[action]
KeyError: 'show_disassembly'
```

**Comment by ratijas** [comment]
Can confirm.

Sublime Text 4147, latest packages.

**Comment by daveleroy** [comment]
Closed in 0.10.0
